docker/README.md

```diff
@@ -23,14 +23,15 @@ The following environment variables can be supplied. Except when noted, it is po
 | Variable | Description |
 | -------- | ----------- |
 | NETWORK        | Leave blank for a private network, otherwise specify one of mainnet, betanet, testnet, or devnet. Only used during a data directory initialization. |
-| FAST_CATCHUP   | If set to 1 on a public network, attempt to start fast-catchup during initial config.                                                               |
-| TELEMETRY_NAME | If set on a public network, telemetry is reported with this name.                                                                                   |
+| PROFILE        | If set, initializes the config.json file according to the given profile. |
```

Comment:
actually maybe I don't understand algocfg profile too well. What values does this variable accept and where are the profiles defined?

Reply:
The profiles are internal to `algocfg`:
```
~$ algocfg profile list
development    Build on Algorand.
participation  Participate in consensus or simply ensure chain health by validating blocks.
conduit        Provide data for the Conduit tool.
relay          Relay consensus messages across the network and support catchup.
```

I have a related PR that improves the `algocfg` error message to print valid profiles if you provide an invalid one: https://github.com/algorand/go-algorand/pull/5324

Reply:
oh this is pretty neat. I didn't know we could set nodes up this way: https://developer.algorand.org/articles/whats-new-in-algorand-3150/#node-profiles

Reply:
mentioning some examples of algocfg profiles in the README here might be helpful.

Reply:
I considered it, but decided to avoid adding content that would have to be updated.

In the future, I think it would be good to update the Dockerfile to take even more inspiration from the postgres container. It lets you run the server and utilities like `psql`. That would help us here because the documentation could say `See available options by running algocfg with "docker run algorand/algod algocfg profile list"`

Reply:
sounds good.